src/logger/mod.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2019 Intel Corporation.
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+use log::LevelFilter;
+
+use std::sync::Mutex;
+
+pub(crate) struct Logger {
+    output: Mutex<Box<dyn std::io::Write + Send>>,
+    start: std::time::Instant,
+}
+
+impl Logger {
+    pub(crate) fn init() {
+        // Default to Debug.
+        // TODO: Accept a command line argument to set the log level.
+        let log_level = LevelFilter::Debug;
```

Comment:
We can have this as a parameter for the init function, and have a default for it (which can be LevelFilter::Debug). This would make it easier to extend it.